PageObject/Buttons/IHaveBurgerButton.cs

```diff
@@ -0,0 +1,13 @@
+using OpenQA.Selenium;
+
+namespace PageObject.Buttons;
+
+public interface IHaveBurgerButton
+{
+    public static IWebElement FindBurgerButton(IWebDriver driver)
+    {
+        var burgerButtonBy = By.Id("react-burger-menu-btn");
+
+        return driver.FindElement(burgerButtonBy);
+    }
+}
```

Comment:
Everywhere: don't create interface for each locator and replace by into find element